src/main/java/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package db;
```

Comment:
```suggestion
package core.basesyntax.db;
```
all your packages should be inside `core.basesyntax`

Reply:
Is folder "resources" not a package?

Reply:
resources is not a package